### ISSUE TITLE
Flagged attributes in a proof request aren't identifiable if viewed by a person with vision impairment

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#1480]
### Description of feature / user story
Flagged attributes in a proof request aren't identifiable if viewed by a person with vision impairment

### Acceptance Criteria
- Flagged attributes should be focusable and identified by a screen reader

### Wireframes or relevant image assets / links
